config/odp-linux-generic.conf

```diff
@@ -31,6 +31,19 @@ pktio_dpdk: {
 	}
 }
 
+# netmap pktio options
+pktio_netmap: {
+	# Default options
+	nr_rx_slots = 0
+	nr_tx_slots = 0
+
+	# Interface specific options
+	eth0: {
+		nr_rx_slots = 0
+		nr_tx_slots = 0
+	}
```

Comment:
Since these options are only significant for virtual interfaces I would simplify the config and make it clear that these options are only relevant for virtual interfaces. It's probably enough to have a single value for all virtual interfaces, so the interface specific options are not needed.

E.g.
```
pktio_netmap: {
	virt: {
		nr_rx_slots = 0
		nr_tx_slots = 0
	}
}
```